composables/storeFinder/usePurchaseLocation.ts

```diff
@@ -0,0 +1,54 @@
+import { removeClientCookie, storeClientCookie } from "../helpers/cookieHelper"
+import { useState, useNuxtApp } from "#app"
+import { Location } from "~~/server/types/GraphQL"
+import { getSpLocations } from "~~/gql/queries/spLocations"
```

Comment:
use @ wherever is possible for consistency purpose

Reply:
Done